source/android/Foundational_Components_Kernel.rst

```diff
@@ -132,30 +132,130 @@ new drivers should always be added as **modules**.
+         _TI_MODULE_OUTS = [
+             # keep sorted
+             "crypto/af_alg.ko",
+             "crypto/algif_hash.ko",
 
-#. In the ``_TI_MODULE_OUTS`` array, add the path to your new kernel module.
+   #. In the ``_TI_MODULE_OUTS`` array, add the path to your new kernel module.
 
-#. Rebuild the kernel as documented in :ref:`android-build-kernel`.
+   #. Rebuild the kernel as documented in :ref:`android-build-kernel`.
 
-#. If the driver module needs to be loaded early (in the ramdisk), edit
-   :file:`${YOUR_PATH}/ti-aosp-15/device/ti/am62x/BoardConfig-common.mk`
-   and add the path to your module:
+   #. If the driver module needs to be loaded early (in the ramdisk), edit
+      :file:`${YOUR_PATH}/ti-aosp-15/device/ti/am62x/BoardConfig-common.mk`
+      and add the path to your module:
 
-   .. code-block:: make
+      .. code-block:: make
 
-      BOARD_VENDOR_RAMDISK_KERNEL_MODULES += \
-              device/ti/am62x-kernel/kernel/$(TARGET_KERNEL_USE)/your_module.ko
+         BOARD_VENDOR_RAMDISK_KERNEL_MODULES += \
+                 device/ti/am62x-kernel/kernel/$(TARGET_KERNEL_USE)/your_module.ko
 
-#. Finally, rebuild the Android images.
+   #. Finally, rebuild the Android images.
 
+********************
+Device tree overlays
+********************
+
+Mapping ``adtbo_idx`` with filenames
+====================================
+
+Device tree overlays can be used to configure additional hardware peripherals.
+These overlays are stored in the :file:`dtbo.img`. This image is generated when
+building the Android kernel as documented in :ref:`android-build-kernel`.
+
+As listed in :ref:`android-dtbo`, we can configure an overlay to be applied
+from U-Boot by setting the ``adtbo_idx`` variable.
+
+To view how the ``adtbo_idx`` maps with the dtbo file, we can inspect the :file:`BUILD.bazel`
+from the `kernel source code <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/BUILD.bazel?h=ti-android-linux-6.6.y#n953>`__.
+Looking at the ``kernel_images()`` macro, we can see:
+
+.. code-block::
+
+   kernel_images(
+    name = "ti_images",
+    build_dtbo = True,
+    build_initramfs = True,
+    dtbo_srcs = [
+        ":ti/k3-am62x-sk-hdmi-audio.dtbo",
+        ":ti/k3-am62x-sk-csi2-ov5640.dtbo",
+        ":ti/k3-am62x-sk-csi2-tevi-ov5640.dtbo",
+        ":ti/k3-am625-sk-microtips-mf101hie-panel.dtbo",
+        ":ti/k3-am62x-sk-lpm-wkup-sources.dtbo",
+        ":ti/k3-am62-lp-sk-microtips-mf101hie-panel.dtbo",
+        ":ti/k3-am625-beagleplay-csi2-ov5640.dtbo",
+        ":ti/k3-am625-beagleplay-csi2-tevi-ov5640.dtbo",
+        ":ti/k3-am625-beagleplay-lincolntech-lcd185-panel.dtbo",
+        ":ti/k3-am62p5-sk-mcan.dtbo",
+        ":ti/k3-am62p5-sk-microtips-mf101hie-panel.dtbo",
+        ":ti/k3-am625-sk-m2-cc3301.dtbo",
+        ":ti/k3-am62p5-sk-m2-cc3301.dtbo",
+        ":ti/k3-am625-sk-wl1837.dtbo",
+
+The ``dtbo_srcs`` array order dicates the index. For example:
+
+.. list-table::
+   :widths: 16 16
```

Comment:
`widths` is a percentage of the current line. `16 16` is equivalent to `50 50` as there are only 2 blocks, but it is just more confusing.

https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table

Reply:
I was not aware of that. Thanks. I removed the `:widths` from v2.